libs/testing/cypress/node/src/lib/plugins/db.ts

```diff
@@ -47,39 +55,67 @@ export async function validateOrg(orgName: string) {
+}
+
+export function deleteOrg(orgId: string) {
+  return db.doc(`orgs/${orgId}`).delete();
+}
+
+//* USERS + ORGS
+
 export async function acceptUserInOrg(userEmail: string) {
   const userQuery = await db.collection('invitations').where('fromUser.email', '==', userEmail).get();
   const [invitation] = userQuery.docs;
   const { id: invitationId } = invitation.data();
   return db.collection('invitations').doc(invitationId).update({ status: 'accepted' });
 }
 
-export async function getRandomOrgAdmin(orgId: string) {
+export async function getRandomOrgUser(orgId: string, type: 'admin' | 'member') {
   const permissionsRef = await db.doc(`permissions/${orgId}`).get();
   const permissions = createPermissions(permissionsRef.data());
-  const adminIds = Object.keys(permissions.roles).filter(userId => {
-    return (
-      permissions.roles[userId] === 'superAdmin' ||
-      permissions.roles[userId] === 'admin'
-    );
-  });
-  const randomIndex = Math.floor(Math.random() * adminIds.length);
-  const adminId = adminIds[randomIndex];
-  const adminRef = await db.doc(`users/${adminId}`).get();
-  return createUser(adminRef.data());
+  const userIds = Object.keys(permissions.roles).filter(uid =>
+    type==='member' ? permissions.roles[uid] === 'member' : permissions.roles[uid] === 'superAdmin' || 'admin');
+  const randomIndex = Math.floor(Math.random() * userIds.length);
+  const userId = userIds[randomIndex];
+  const userRef = await db.doc(`users/${userId}`).get();
+  return createUser(userRef.data());
 }
 
-export function deleteUser(userId: string) {
-  return db.doc(`users/${userId}`).delete();
+//* ORGS + MOVIES
+
+async function getOrgMovies(orgId: string) {
+  const movies = [];
+  const userQuery = await db.collection('movies').where('orgIds', 'array-contains', orgId).get();
+  userQuery.forEach(doc => movies.push(doc.data()));
+  return movies.filter(movie => movie.title.international || movie.title.original);
 }
 
-export async function getOrgByName(orgName: string) {
-  const userQuery = await db.collection('orgs').where('denomination.full', '==', orgName).get();
-  const [org] = userQuery.docs;
-  if(!org) return null;
-  return createOrganization(org.data());
+//* ORGS + USERS + MOVIES
+
+export async function getRandomScreeningData(data: { app: App, access: ModuleAccess, userType: 'admin' | 'member', moviesWithScreener?: boolean}) {
+  const { app, access, userType, moviesWithScreener } = data;
+  let result = {};
+  do {
+    const org = await getRandomOrg({app, access, userType});
+    const user = await getRandomOrgUser(org.id, userType);
+    let movies = (await getOrgMovies(org.id))
+      .filter(movie => movie.title.international && movie.app.festival.status == "accepted" && movie.app.festival.access == true);
+    if(!isUndefined(moviesWithScreener)) movies = movies.filter(movie => moviesWithScreener ? movie.promotional.videos.screener.jwPlayerId : !movie.promotional.videos.screener.jwPlayerId);
```

Comment:
```suggestion
    const jwPlayerId = movie.promotional.videos.screener.jwPlayerId;
    movies = movies.filter(movie => moviesWithScreener ? jwPlayerId : true);
```

looks like that `moviesWithScreener` is always provided, so no need to use `isUndefined`

Reply:
I do need the `isDefined`, because in a test, I retrieve all the movies just as the Screening tab does, and check if all are available in `mat-options` :
![image](https://user-images.githubusercontent.com/78768220/162910795-c105a068-4078-444a-8634-9eaecf1119e5.png)